.eslintignore

```diff
@@ -0,0 +1,2 @@
+# /node_modules/* and /bower_components/* in the project root are ignored by default
+webapp
```

Comment:
I added it because We were linting `webapp/node_modules`. 

If we add it to the script. We need to lint spec, scripts, etc. too not only src.

Using an ignore file give us the possibility to exclude whatever we want.

What do you think?